.github/workflows/on-push.yml

```diff
@@ -317,6 +317,14 @@ jobs:
           cd content
           demisto-sdk pre-commit --validate --show-diff-on-failure --verbose -i Packs/HelloWorld -i Packs/QRadar/Integrations/QRadar_v3 --mode=nightly
 
+      - name: run pre-commit with default pre-commit config shipped with the SDK
+        timeout-minutes: 60
+        run: |
+          source $(poetry env info --path)/bin/activate
+          cd content
```

Comment:
what repo is this?

Reply:
content repo after deleting the original .pre-commit-config_template.yaml

would like to test it out on some random repo?

Reply:
i think this is fine unless Anar says otherwise

Reply:
keeping it as it is